src/libs/sfdk/vmconnection.cpp

```diff
@@ -660,7 +662,11 @@ bool VmConnection::vmStmStep()
         } else if (m_lockDownRequested) {
             // noop
         } else if (m_connectRequested) {
-            if (m_connectOptions & VirtualMachine::AskStartVm) {
+            if (shouldCheckSshPortIsOccupied()
+                    && isPortOccupied(m_vm->sshParameters().port())) {
+                ui()->warn(Ui::SshPortOccupied);
+                vmStmTransition(VmStartingError, "ssh port is occupied");
```

Comment:
I would still prefer omitting 'is' here to be in line with other messages in this context :) And looking at it again, ssh would be better upper case

```suggestion
                vmStmTransition(VmStartingError, "SSH port occupied");
```

Reply:
Updated.